mem/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "mem"
```

Comment:
Hmm, not sure I love that name, sounds a bit like a pathname! How about, as mentioned above, you move it to a `parity-crypto` subcrate and so we publish it as part of that? If you don't like `memzero`, then just `mem-utils` seems like it could work.